README.md

```diff
@@ -1103,14 +1103,20 @@ column type.
 #### Convert a column from text to json
 
 Postgres' `alter column` command will not automatically convert a `text`
-column to `json`, but it can still be done with plain SQL.
+column to `json`, but it can still be done with a piece of plain SQL.
 
-```sql
-alter table versions
-alter column object type json
-using object::json;
+```ruby
+def up
+  change_column :versions, :object, 'jsonb USING object::jsonb'
+end
+
+def down
+  change_column :versions, :object, 'text USING object::text'
+end
 ```
 
+You'll have to destroy or migrate all `versions` data before running this migration.
```

Comment:
You can still migrate from YAML to JSON by doing something like this:
```ruby
add_column :versions, :object, :new_object, :jsonb

PaperTrail::Version.find_each do |version|
  version.update_column :new_object, YAML.load(version.object)
end

remove_column :versions, :object
rename_column :versions, :new_object, :object
```

Of course it depends on how much data you have to migrate, and whether downtime of any kind is an option. If it's not, but it's okay for any UI in your app to temporarily no longer have any version object data, you can do this:
```ruby
rename_column :versions, :object, :old_object
add_column :versions, :object, :jsonb
```

That will run instantaneously, and any new version objects will use the new JSONB column. Then you can run a script in the background that moves the YAML data from the old column into the new column. Then, you can remove the old column.

@jaredbeck perhaps :arrow_double_up: should be part of the documentation. I can create a new PR after this one is merged if you'd like.